x/pairing/keeper/keeper.go

```diff
@@ -30,6 +30,16 @@ type (
 	}
 )
 
+// sanity checks at start time
+func init() {
```

Comment:
where is this called?

Reply:
`init()` function is a feature of Go (https://go.dev/ref/spec#Package_initialization)
If it exists (per package) then it is called at package "startup" before `main().`